mscore/propertymenu.cpp

```diff
@@ -464,7 +468,7 @@ void ScoreView::elementPropertyAction(const QString& cmd, Element* e)
             }
       else if (cmd == "style") {
             EditStyle es(e->score(), 0);
-            es.setPage(EditStyle::PAGE_NOTE);
```

Comment:
setPage is only used here, so you should disable it [here](https://github.com/musescore/MuseScore/blob/master/mscore/editstyle.cpp#L1198) and [here](https://github.com/musescore/MuseScore/blob/master/mscore/editstyle.h#L93).

Reply:
I don't know, this may be a useful function in the future. There's no harm in leaving it exposed, I think.

Reply:
True, but won't the compiler complain about an unused method? Maybe comment it out (to save Jojo the trouble ;)).

Reply:
I don't think so... only unused variables, I thought.

Reply:
It wanrts about unused statuc funtions too